### PR TITLE
Refactor js

### PR DESCRIPTION
**Pull request by shortdiv**
Removed unnecessary code and streamlined libraries into a separate helper file, for global helper functions.

Refactored JS so that it is more modular.

### CODE REVIEW

**Comment by shortdiv** [comment]
ping @JoeGermuska @zachwise @rapoulson

**Comment by JoeGermuska** [comment]
This looks like win to me. Always better to have less code. Great to see that `lodash` solves the class inheritance question smoothly. Agree with minimizing globals/pushing stuff out of the module into `this`/`window` scope.

We should let @zachwise absorb all of this before merging, but I think this is a good move towards simpler and more modern JS...

**Comment by zachwise** [comment]
Just looking at this now. I'm not totally sure what `lodash` is for but it adds around 500k to the JS file which seems excessive to me. Why is this better than what we had? Is it worth 500k? This is why I don't like relying on 3rd party libraries.

**Comment by shortdiv** [comment]
[Lodash](https://lodash.com/) is a js utility library that adds methods we can use to mutate strings, arrays and objects. It's useful because it's a catch all so we don't have to create our own functions or include multiple lib and util files for methods. 

You do make a good point about file size, I believe we can require the methods we actually use and that method's dependencies to save on file size on lodash.

Note to self: [lodash function dependencies](https://github.com/lodash/lodash-cli/blob/aa473983607a39351ba6c5c7f2375aadfd39f880/lib/mapping.js#L379-L917)

File: src/js/compositions.js [hunk 1]
```diff
@@ -95,39 +89,49 @@ KL.Pullquote = (function() {
...
-        this.options.height = window.innerHeight;
-    }
-
     /*	INIT
     ================================================== */
-    this.load_quotes();
-
+    load_quotes();
 
 })();
```
Comment (shortdiv) [inline src/js/compositions.js:137]: Removed the call to `this` so as not to pollute the window object. Assuming you're in the scope of the function, variables and functions should be accessible within it.

**Pull request merged**